template/javascript/default/src/components/AppFooter.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <v-footer height="40" app>
+  <v-footer
+    height="40"
```

Comment:
Same here, a vuetify eslint config setting. I think it's 4 for the docs, we can mimic that.